index.html

```diff
@@ -1430,9 +1441,48 @@
 
 ## Harassment
 
-<aside class="issue">
-  This section has not been written.
-</aside>
+Online <dfn>harassment</dfn> is the "pervasive or severe targeting of an individual or group online
+through harmful behavior" [[PEN-Harassment]]. Harassment is a prevalent problem on the Web,
+particularly via social media. Harassment may be more severe and its consequences more impactful for
+LGBTQ people and women [[?PEW-Harassment]].
+
+<a>Harassment</a> is both a violation of privacy itself and can be a consequence of other violations
+of privacy. 
+
+Abusive online behavior may include: sending <a href="#unwanted-information">unwanted
+information</a>; directing others to contact or bother a person ("dogpiling"); disclosing sensitive
```

Comment:
Nitpick but "unwanted information" should be a `dfn` and this could reference it.